scripts/update-drafts.ts

```diff
@@ -35,9 +35,14 @@ async function main() {
   const compat = new Compat();
 
   // Build a set of used BCD keys.
-  const usedFeatures = new Set<string>(
-    Object.values(features).flatMap((data) => data.compat_features)
-  );
+  const webFeatures = new Map<string,string>();
+  Object.values(features).map((data) => {
+    if(data.compat_features){
+    for(const compatFeature of data.compat_features){
+      webFeatures.set(compatFeature, data.name);
```

Comment:
Anticipating that we might solve https://github.com/web-platform-dx/web-features/issues/1173, can you throw an error here if there is overlap?

```js
if (compatFeatureToName.has(compatFeature)) {
  throw new Error(`${compatFeature} seen in multiple features`);
}
```

or similar.